frontend/src/components/chatapp/SearchBar.tsx

```diff
@@ -0,0 +1,60 @@
+// components/SearchBar.js
+import React, { useState } from 'react';
+import { DocumentData, collection, getDocs, query, where } from "firebase/firestore";
+import { db } from '../../../../backend/firebase/firebase';
+import ConversationItem from './ConversationItem';
+import magnifyglass from '../../assets/magnifyglass'
+
+const SearchBar = ({ onSearch }) => {
+  const [searchTerm, setSearchTerm] = useState("");
+  const [user, setUser] = useState<(DocumentData)>();
+  const [err, setErr] = useState(false);
+
+
+  const handleSearch = async () => {
+    const q = query(collection(db, "ChatUserPsychiatrists"), where("name", "==", searchTerm))
+    const querySnapshot = await getDocs(q);
+
+    try {
+      querySnapshot.forEach((doc) => {
+        setUser(doc.data());
+        // console.log(user);
+      });
+    } catch (err) {
```

Comment:
unclear what purpose of err state is

Reply:
if for debugging, should remove, if for displaying errors to users, should change code to reflect this in catch block